docs/2_getting_started_session.md

```diff
@@ -220,15 +220,7 @@ dds user add [Email address] --role "Researcher" --project "<Project ID>"
 
 > Another command to invite researchers is `dds project access grant --project "<Project ID>" --email "<email>" `
 
-###### Sub-section taks
-
- - [ ] TASK: Release the project after making sure all the data was uploaded, remember to specify the correct project ID
- 
- - [ ] TASK: Download the project contents, and verify that they are the correct. You can download data with your unit admin account.
-
- - [ ] Optional: If you have another email address, use your Unit Admin / Personnel account to invite that email as a researcher. Once you have registered the "researcher" account, authenticate using the researcher account credentials and use the dds data get command as described above to download the data.
-
-## Optional task - DDS Cli Token
+##### DDS Cli Token
```

Comment:
I feel like it is quite an advance feature. And probably most people won't even need to use it never

Reply:
Yup exactly, don't we want it to be very optional?

Reply:
The task is marked as optional. Or you mean the whole section has optional?

Reply:
Maybe the whole section - i.e. if it's very advanced, then we probably don't want everyone to have to even read the section.

Reply:
okay, done

Reply:
I kind of disagree that it is so advanced and never going to be used. It is very useful in my opinion, in particular because units have no other option to deliver data between themselves other than inviting each other as Researchers. Then this option can be very useful for them. Also, I always saw this workshop as way to show them such fine features, apart from giving them the big picture

Reply:
I don't mind, actually. 
But I don't know how common do they have to deliver data between themselves. But as Aishling is not here today. I can move it back again to get approved from you, and then you can add the things that got deleted from your changes and Monday next week we finish the discussion

Reply:
My only opinion is that the task is optional, then the text should be optional - whether it should all be optional or not, I'll leave up to you guys! Go ahead and make whichever call.